src/hooks/useOnyx.ts

```diff
@@ -0,0 +1,74 @@
+import {useMemo} from 'react';
```

Comment:
Hey @allgandalf and @luacmartins, do you have any suggestions on how to avoid using 'any' in this hook wrapper? I've attempted to modify the structure, but so far this is the most successful outcome.

Reply:
Maybe something like this? https://github.com/Expensify/App/blob/4fa2cdfd3f0ef7950bb40168681cff1a9e643e6f/src/types/onyx/SearchResults.ts#L391-L397